ci-operator/step-registry/cloud-maintenance/OWNERS

```diff
@@ -0,0 +1,5 @@
+approvers:
+- cspi-qe-ocp-lp
+options: {}
+reviewers:
+- cspi-qe-ocp-lp
```

Comment:
Can you please tell, those are the only reviewers we are required?

Reply:
I put our alias there because we will have to maintain this. If others want to use/maintain the script as well, we can add them. For now, it will just be us.